docs/career-and-open-positions/senior-software-engineer.md

```diff
@@ -1,48 +1,65 @@
 # Senior Software Engineer
 
-## **About Airbyte**
+Join [Airbyte's](https://airbyte.io/) small, but growing, engineering team. We are backed by the world's [top investors](https://docs.airbyte.io/career-and-open-positions#our-investors). 
 
-[Airbyte](https://airbyte.io/) is the upcoming [open-source](https://github.com/airbytehq/airbyte) standard for EL\(T\). We enable data teams to replicate data from applications, APIs, and databases to data warehouses, lakes, and other destinations. We believe open-source is the only approach that can solve the problem of data integration. It allows us to cover the long tail of integrations while enabling teams to adapt pre-built connectors to their needs.
+You will spend your time solving one of the most complex and unsolved problem with data: Access. We're on a mission to allow anyone to access data, wherever it is, and make it available into the system of their choice.
 
-Airbyte is remote friendly, with most of the team based in the Bay Area. We’re fully open as a company. Our [company handbook](https://docs.airbyte.io/company-handbook), [culture & values](https://docs.airbyte.io/company-handbook/culture-and-values), [strategy](https://docs.airbyte.io/company-handbook/strategy) and [roadmap](https://docs.airbyte.io/roadmap) are open to all.
+We are passionate about data, open source, our community and scale! 
 
-We're a seed-stage company backed by some of the world's [top investors](https://docs.airbyte.io/career-and-open-positions#our-investors) and believe in product-led growth, where we build something awesome and let our product bring the users, rather than an outbound sales engine with cold calls.
+We love Java, Python, Node, K8s, Docker... but in the end, it doesn't matter: we just want to build the best product.
 
-## **Responsibilities**
+We believe in product-led growth, not cold sales calls! We build something awesome, and we let our product bring the users.
```

Comment:
```suggestion
```

I'd delete this line, it makes it sound like we aren't building the awesome product based off of user demand.